docs/en/connector-v2/sink/Http.md

```diff
@@ -36,7 +36,6 @@ They can be downloaded via install-plugin.sh or from the Maven central repositor
 |-----------------------------|--------|----------|---------|-----------------------------------------------------------------------------------------------------|
 | url                         | String | Yes      | -       | Http request url                                                                                    |
 | headers                     | Map    | No       | -       | Http headers                                                                                        |
-| params                      | Map    | No       | -       | Http params                                                                                         |
```

Comment:
Why remove this?

Reply:
Because httpsink did not use the param parameter, the specific method diagram was shown in the previous comment @Hisoka-X

Reply:
Seem like this is a feature we don't supported. Could you open an issue for this feature?  Also please remove it from option rules https://github.com/apache/seatunnel/blob/7cbaedf3190cc33a637c4668f50558f2fe3ffe66/seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/sink/HttpSinkFactory.java#L39

Reply:
ok，i will open issue for this feature . This time, the httpsink parameter has been modified and e2e httpsink test case support has been added to cover the code